chipsec/utilcmd/mem_cmd.py

```diff
@@ -128,6 +129,17 @@ def run(self):
             (va, pa) = self.cs.mem.alloc_physical_mem( size )
             self.logger.log( '[CHIPSEC] Allocated %X bytes of physical memory: VA = 0x%016X, PA = 0x%016X' % (size, va, pa) )
 
+        elif 'search' == op and len(self.argv) > 5:
+            phys_address = int(self.argv[3],16)
+            size         = int(self.argv[4],16)
+            target = 0
+            for i in range(0, len(self.argv[5])):
+                target += ord(self.argv[5][i]) << (i * 8)
+                      
+            buffer = self.cs.mem.read_physical_mem( phys_address, size )
+            offset = buffer.find(self.argv[5])
```

Comment:
Should handle the case where the search string is not found and -1 is returned.

Reply:
Agree.